src/defaults.js

```diff
@@ -0,0 +1,102 @@
+"use strict";
+const { defaultOptions } = require("./utils");
+const { sign } = require("tweetnacl");
+const { toByteArray } = require("base64-js");
+
+BigInt.prototype.toJSON = function () {
+  return this.toString();
+};
+
+const defaultBaseOptions = {
+  APIKey: "",
+  skynetApiKey: "",
+  customUserAgent: "",
+  customCookie: "",
+  onDownloadProgress: undefined,
+  onUploadProgress: undefined,
+  loginFn: undefined,
+};
+
+const defaultGetEntryOptions = {
+  ...defaultBaseOptions,
+  endpointGetEntry: "/skynet/registry",
```

Comment:
```suggestion
  ...defaultOptions("/skynet/registry"),
```

Same below in `defaultSetEntryOptions`.